src/QueryRenderer.ts

```diff
@@ -146,6 +172,7 @@ class QueryRenderChild extends MarkdownRenderChild {
     }: {
         tasks: Task[];
         content: HTMLDivElement;
+        app: App;
```

Comment:
is this actually used?

Reply:
I don't believe it is used. I missed this.